doc/ko/uftrace-report.md

```diff
@@ -37,6 +44,11 @@ REPORT 옵션
 
 \--task
 :   함수의 통계자료가 아닌 태스크를 요약해서 보고한다.
+    -f 옵션을 이용해 출력 필드를 사용자가 지정할 수 있다.
+    가능한 값들로는: `total`, `self`, `func` 그리고 `tid`가 있다.
+    여러 필드를 갖는 경우 콤마로 구분된다. 모든 필드를 감추기 위한
+    (단일하게 사용되는) 'none' 특수 필드가 있으며 기본적으로 'total,self,func,tid' 가 사용된다.
+    상세한 설명은 *FIELDS* 를 참고한다.
 
```

Comment:
original:

```
\--task
:   Report task summary information rather than function statistics.
    Customize field in the output with -f option. Possible values are: `total`,
    `self`, `func` and `tid`.  Multiple fields can be set by using comma.
    Special field of 'none' can be used (solely) to hide all fields.
    Default is 'total,self,func,tid'.  See *TASK FIELDS*.
```